webrtc.html

```diff
@@ -6597,6 +6599,10 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
       <p class="issue">How do you know when it has stopped? This seems
       like an SDP question, not a media-level question. (Suggestion: when the
       track is ended, either through port 0, or removing the a=msid attrib)</p>
+
+      <p>When a <a>remote source</a> is notified that a <code>
+      <a>MediaStreamTrack</a></code>, using the source, has <code>ended</code>
+      [[!GETUSERMEDIA]] it must TBD.</p>
```

Comment:
"it MAY choose to stop using resources on the incoming stream, for instance by turning off the decoder."
Do we want to say anything about the possibility of negotiating it away in the next offer/answer (state sendonly or inactive for the media section)?